### DISCUSSION TITLE
center and zoom

### DISCUSSION THREAD

**Discussion by limzw** [Mapsui/Mapsui#2135] [Q&A]
Hi all, i would like to center on and zoom in on a new point on the map. I am using the OSM. Would appreciate it if someone can kindly advise me on this!

**Comment by Smurfj3** [comment]
MapView.Map.Navigator.CenterOnAndZoomTo();

**Reply by ztarbug** [reply]
Method doc can be found here: https://mapsui.com/api/Mapsui.Navigator.html#Mapsui_Navigator_CenterOnAndZoomTo_Mapsui_MPoint_System_Double_System_Int64_Mapsui_Animations_Easing_

Is there any doc, that explains which coordinate system MPoint is referring to? Doc just says x,y... Thanks!

**Reply by pauldendulk** [reply]
That depends on the coordinate system that is used for the map. If you are using openstreetmap, like most users, it is Spherical Mercator ([EPSG:3857](https://epsg.io/3857)). 

Here is some more information https://mapsui.com/documentation/projections.html
